api/routes/jwt/schema.js

```diff
@@ -1,14 +1,15 @@
 const loginSchema = {
   body: {
     type: 'object',
-    required: ['email', 'password'],
     properties: {
-      email: {
+      username: {
```

Comment:
is there a more stricter checking for this? like auto stripping the input and checking for empty values?

Reply:
Use joi